src/rollup-plugin-dependency-info.ts

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs';
+import path from 'path';
+import {OutputOptions, OutputBundle} from 'rollup';
+
+type Dependency = {
+  fileName: string;
+  size: number;
+  delta?: number;
+};
+type Dependencies = {
+  [filePath: string]: Dependency;
+};
+type DependencyType = 'direct' | 'shared';
+export type DependencyInfoOutput = Record<DependencyType, Dependencies>;
+
+export function rollupPluginDependencyInfo(cb: (dependencyInfo: DependencyInfoOutput) => void) {
+  let outputDir: string;
+  let cache: {
+    [fileName: string]: number;
+  } = {};
+  let output: DependencyInfoOutput = {
+    direct: {},
+    shared: {},
```

Comment:
nit: can we use `common` here instead of `shared`? I'd love to keep the naming variations to a minimum, and since our rollup config is already using `web_modules/common` it would be nice to stick to that.